cogeo_mosaic/backends/stac.py

```diff
@@ -0,0 +1,101 @@
+"""cogeo-mosaic STAC backend."""
+
+import functools
+import json
+from typing import Any, Callable, Dict, List, Optional
+
+import mercantile
+import requests
+
+from cogeo_mosaic.backends.base import BaseBackend
+from cogeo_mosaic.backends.utils import get_assets_from_json
+from cogeo_mosaic.mosaic import MosaicJSON
+
+
+def default_stac_accessor(feature: Dict):
+    """Return specific feature identifier."""
+    link = list(filter(lambda link: link["rel"] == "self", feature["links"]))[0]
```

Comment:
is this part of the spec?

Reply:
Self link is strongly recommended but not required.  You could also build the item URL from `{base_url}/collections/{collectionID}/items/{itemID}` if self link is missing.

Reply:
I'll add something. 
Any way I think this MUST be covered by the documentation on the `accessor` option